src/mctpd.c

```diff
@@ -3074,9 +3074,10 @@ static int setup_testing(ctx *ctx) {
 
 static void print_usage(ctx *ctx)
 {
-	fprintf(stderr, "mctpd [-v] [-N]\n");
+	fprintf(stderr, "mctpd [-v] [-N] [-t usecs]\n");
 	fprintf(stderr, "      -v verbose\n");
 	fprintf(stderr, "      -N testing mode. Not safe for production\n");
+	fprintf(stderr, "      -t timeout in usecs\n");
```

Comment:
Timeout for what? This may need to be a little more descriptive.

And why microseconds here? milliseconds may be easier to deal with.